core/test/audits/non-composited-animations-test.js

```diff
@@ -119,6 +119,7 @@ describe('Non-composited animations audit', () => {
       .toBeUndefined();
   });
 
+  // Note: might be safe to delete, seems `failureReasonsMask: 0` no longer appears in traces.
```

Comment:
Not when it's zero though.

Reply:
I did a trace with 1 compositor animation and 1 non-compositor animation and this is what I saw:

```json
{
    "args": {
        "data": {
            "compositeFailed": 0
        }
    },
    "cat": "blink.animations,devtools.timeline,benchmark,rail",
    "id2": {
        "local": "0x11d00107ec8"
    },
    "name": "Animation",
    "ph": "n",
    "pid": 94790,
    "scope": "blink.animations,devtools.timeline,benchmark,rail",
    "tid": 259,
    "ts": 53551489038
},
{
    "args": {
        "data": {
            "compositeFailed": 8224,
            "unsupportedProperties": [
                "left"
            ]
        }
    },
    "cat": "blink.animations,devtools.timeline,benchmark,rail",
    "id2": {
        "local": "0x11d001081f8"
    },
    "name": "Animation",
    "ph": "n",
    "pid": 94790,
    "scope": "blink.animations,devtools.timeline,benchmark,rail",
    "tid": 259,
    "ts": 53551489046
},
```

I see `compositeFailed: 0` for the compositor animation and `compositeFailed: 8824` for the non-compositor animation.

Reply:
What was the page?

The trace that is checked in for this PR from the animations fixtures (which was the source of the original trace, I believe) no longer has a `"compositeFailed": 0`.

Reply:
You linked to the existing trace. In the trace from this PR, which I just re-generated, there is no `compositeFailed: 0`.

I just tried your given HTML and I see it now. I suppose the problem is that the `/perf/animations.html` fixture is not what was used to create `animations.json`, like I assumed.

Reply:
Ah, there's been some changes to the file since the trace was collected. That explains why `font-size` is now included in the failed properties (it was added after the trace was collected). But even using the original file, still no `compositeFailed: 0`.

Reply:
I dug around in the current `perf/animations.html` file, and it turns out the the compositor animation (which would have emitted an event with `compositeFailed: 0`) doesn't run at all because I used a `,` instead of a `;` in the CSS

https://github.com/GoogleChrome/lighthouse/blob/f6171e018bd17977ad276cdaceb6765e927e7c6c/cli/test/fixtures/perf/animations.html#L22

🙃 🙃 🙃